skt/reporter.py

```diff
@@ -51,6 +51,67 @@ def gzipdata(data):
     return tstr.getvalue()
 
 
+def load_state_cfg(current_cfg, statefile):
+    """Load state information from a state file.
+
+    It takes the current config and adjusts state information within the
+    config based on the state file provided.
+
+    Args:
+        current_cfg:    Existing cfg dict to adjust
+        statefile:      Path to a skt state file to read
+
+    Returns: A cfg dictionary.
+    Raises: Exception if required 'runner' section is missing
+
+    """
+    cfg = current_cfg
```

Comment:
I don't think this is a good idea. We want to read a clean new state each time, not add it to previous ones. Checking out reporting with more state files to report, I get the first file passed reported multiple times, instead of having each of them reported once.